item-spec/common-metadata.md

```diff
@@ -9,6 +12,18 @@
 Various *examples* are available in the folder [`examples`](examples/).
 *JSON Schemas* can be found in the folder [`json-schema`](json-schema/).
 
+## Basics
+
+Descriptive fields to give a basic overview of a STAC Item.
+
+- [JSON Schema](json-schema/basics.json)
+
+| Field Name  | Type   | Description                                                  |
+| ----------- | ------ | ------------------------------------------------------------ |
+| title       | string | A human readable title describing the Item. |
+| description | string | Detailed multi-line description to fully explain the Item. [CommonMark 0.29](https://commonmark.org/) syntax MAY be used for rich text representation. |
```

Comment:
Adding the description field is an addition which should be put into the changelog.

Reply:
I added the description and a little bit more about the common metadata basics fields to the changelog, let me know if it's overkill

Reply:
Some of the entries will probably be removed once the changelog is merged for 0.9 as these are no changes between 0.8.x and 0.9. But we can handle that later, I guess.